contents/javascript/Javascript-framework-React.md

```diff
@@ -0,0 +1,142 @@
+<frontmatter>
+  title: React
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# React
+
+Author: [Aadyaa Maddi](https://github.com/amad-person)
+
+Reviewers: [Amrut Prabhu](https://github.com/amrut-prabhu), [Marvin Chin](https://github.com/marvinchin)
+
+### What is React?
+
+**React** is an open-source JavaScript <tooltip content="React only handles the view component of your application. It is not an MVC framework!">library</tooltip> for building composable user interfaces. It was developed by Facebook and released to the world in 2013, and is now used in a huge number of popular applications, including Facebook and Instagram.
```

Comment:
This tooltip doesn't match the keyword. A reader who knows what a library is will not checkout the tooltip, thinking that the tooltip gives the definition of a library. Perhaps this?
```
**React** is an open-source JavaScript library for <tooltip content="React only handles the UI componeent of your application. It is not a full web application framework!">building the UI of web apps</tooltip>. It was developed by Facebook and released to the world in 2013, and is now used in a huge number of popular applications, including Facebook and Instagram.
```
I left out the _composable_ part as it felt premature to introduce that technical term. In fact, there is no mention of that term anywhere else in the article which is odd given that it was mentioned in the definition of the tool. Omitted MVC for the same reasons; there is no guarantee that the reader will know what MVC is.
huge -> large?

Reply:
Noted, will update!